app/services/serialize_offer.rb

```diff
@@ -0,0 +1,64 @@
+class SerializeOffer
```

Comment:
This service saves an Offer and OfferConditions based on structured parameters (rather than a string list of conditions).

Maybe we call this one `SaveOfferFromParams` and then in the upcoming refactor of `UpdateOfferConditions`, we call that one `SaveOfferFromText` or similar?